git_machete/git_operations.py

```diff
@@ -918,7 +918,10 @@ def do_rebase() -> None:
             try:
                 if not opt_no_interactive_rebase:
                     rebase_opts.append("--interactive")
-                self._run_git("rebase", *rebase_opts, "--empty=drop", "--onto", onto, from_exclusive, branch)
+                if self.get_git_version() < (2, 26, 0):
+                    self._run_git("rebase", *rebase_opts, "--onto", onto, from_exclusive, branch)
+                else:
+                    self._run_git("rebase", *rebase_opts, "--empty=drop", "--onto", onto, from_exclusive, branch)
```

Comment:
I'd just use `rebase_opts.append(...)`, as in line 920 above (instead of repeating all params twice)